core/src/main/scala/matryoshka/CofreeOps.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2014 - 2015 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package matryoshka
+
+import scalaz._
+
+sealed class CofreeOps[F[_], A](self: Cofree[F, A]) {
```

Comment:
Since you aren't subclassing this type, I'd opt for `final` as it is a bit more precise ('sealed' would allow you to subclass the type in this file), unless we know of some benefit `sealed` provides (I don't know of any, but others might).